tests/acceptance/edit-note-test.js

```diff
@@ -31,3 +31,33 @@ moduleForAcceptance('Acceptance | edit note')
       assert.equal(find('.spec-reminder-item').text().trim(), 'Remind me to get a job tomorrow', 'should list updated title on page');
     });
   });
+
+  test('should rollback changes when undo is clicked', function(assert) {
+    visit('reminders/new');
+    fillIn('.spec-input-title', 'Remind me to go to class');
+    fillIn('.spec-input-date', 'Tomorrow');
+    fillIn('.spec-textarea-notes', 'Because');
+    click('.add-notes--submit');
+    andThen(function() {
+      assert.equal(currentURL(), 'reminders/new', 'should route to reminders/new');
+      assert.equal(find('.spec-reminder-item').length, 1, 'should create a note');
+      assert.equal(find('.spec-reminder-item').text().trim(), 'Remind me to go to class', 'should list original note title');
+    });
+
+    click('.spec-reminder-item:first');
+    click('.spec-link-to-edit');
+    andThen(function() {
+      assert.equal(currentURL(), '/reminders/1/edit', 'should route to edit page')
+    });
+
+    fillIn('.spec-input-title', 'Remind me to get a job tomorrow');
+    fillIn('.spec-input-date', 'Tomorrow');
+    fillIn('.spec-textarea-notes', 'Nah');
+    andThen(function() {
```

Comment:
Would you fix the spacing. I would do a space between every assertion, top and bottom. So pretty much all the andThen()